app/components/isp-jam-auth/template.hbs

```diff
@@ -14,6 +14,9 @@
       </div>
       <div class="form-group">
         <button disabled={{disableLogin}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
+          {{#if authenticating}}
+              <i class="fa fa-spinner fa-lg fa-spin"></i>
```

Comment:
There's really not much to review here (as long as the state of the `authenticating` property is handled correctly), but I encourage you to read this page sometime:

http://fontawesome.io/accessibility/

Basically, your font-awesome icon might be invisible to blind people, because there's no reader-friendly equivalent.

See quotes below:
> If you're using an icon to add some extra decoration or branding, it does not need to be announced to users as they are navigating your site or app aurally. Additionally, if you're using an icon to visually re-emphasize or add styling to content already present in your HTML, it does not need to be repeated to an assistive technology-using user. 

But

>  If you're using an icon to convey meaning (rather than only as a decorative element), ensure that this meaning is also conveyed to assistive technologies.